test/Table_Tests/src/In_Memory/Table_Conversion_Spec.enso

```diff
@@ -202,4 +203,68 @@ spec =
+            table = Table.new [["aaa", [1, 2, 3]], ["bbb", values_to_expand], ["ccc", [5, 6, 7]]]
+            expected = Table.new [["aaa", [1, 1, 2, 2, 2, 3]], ["bbb", [10, 11, 20, 21, 22, 30]], ["ccc", [5, 5, 6, 6, 6, 7]]]
+            table.expand_to_rows "bbb" . should_equal expected
+
+        Test.specify "Can expand Pairs" <|
+            values_to_expand = [Pair.new 10 20, Pair.new "a" [30], Pair.new 40 50]
+            table = Table.new [["aaa", [1, 2, 3]], ["bbb", values_to_expand], ["ccc", [5, 6, 7]]]
+            expected = Table.new [["aaa", [1, 1, 2, 2, 3, 3]], ["bbb", [10, 20, "a", [30], 40, 50]], ["ccc", [5, 5, 6, 6, 7, 7]]]
+            table.expand_to_rows "bbb" . should_equal expected
+
+        Test.specify "Can expand Ranges" <|
+            values_to_expand = [Range.new 10 12, Range.new 20 27 step=3, Range.new 30 31]
+            table = Table.new [["aaa", [1, 2, 3]], ["bbb", values_to_expand], ["ccc", [5, 6, 7]]]
+            expected = Table.new [["aaa", [1, 1, 2, 2, 2, 3]], ["bbb", [10, 11, 20, 23, 26, 30]], ["ccc", [5, 5, 6, 6, 6, 7]]]
+            table.expand_to_rows "bbb" . should_equal expected
+
+        Test.specify "Can expand Date_Ranges" <|
+            range0 = (Date.new 2020 02 28).up_to (Date.new 2020 03 01)
+            range1 = (Date.new 2020 10 28).up_to (Date.new 2020 11 16) . with_step Date_Period.Week
+            range2 = (Date.new 2023 07 03).up_to (Date.new 2023 10 03) . with_step Date_Period.Month
+            values_to_expand = [range0, range1, range2]
+            values_expanded = [Date.new 2020 02 28, Date.new 2020 02 29, Date.new 2020 10 28, Date.new 2020 11 4, Date.new 2020 11 11, Date.new 2023 07 03, Date.new 2023 08 03, Date.new 2023 09 03]
+            table = Table.new [["aaa", [1, 2, 3]], ["bbb", values_to_expand], ["ccc", [5, 6, 7]]]
+            expected = Table.new [["aaa", [1, 1, 2, 2, 2, 3, 3, 3]], ["bbb", values_expanded], ["ccc", [5, 5, 6, 6, 6, 7, 7, 7]]]
+            table.expand_to_rows "bbb" . should_equal expected
+
+        Test.specify "Can expand mixed columns" <|
+            values_to_expand = [[10, 11], 22.up_to 26, (Date.new 2020 02 28).up_to (Date.new 2020 03 01)]
+            values_expanded = [10, 11, 22, 23, 24, 25, Date.new 2020 02 28, Date.new 2020 02 29]
+            table = Table.new [["aaa", [1, 2, 3]], ["bbb", values_to_expand], ["ccc", [5, 6, 7]]]
+            expected = Table.new [["aaa", [1, 1, 2, 2, 2, 2, 3, 3]], ["bbb", values_expanded], ["ccc", [5, 5, 6, 6, 6, 6, 7, 7]]]
+            table.expand_to_rows "bbb" . should_equal expected
+
+        Test.specify "Respects `at_least_one_row` param" <|
+            values_to_expand = [[10, 11], [], [30]]
+            table = Table.new [["aaa", [1, 2, 3]], ["bbb", values_to_expand], ["ccc", [5, 6, 7]]]
+            expected = Table.new [["aaa", [1, 1, 2, 3]], ["bbb", [10, 11, Nothing, 30]], ["ccc", [5, 5, 6, 7]]]
+            table.expand_to_rows "bbb" at_least_one_row=True . should_equal expected
```

Comment:
please add also a test case for `at_least_one_row=False` to show it disappearing - I did not see a test for a row disappearing due to empty expanded collection anywhere (or did I miss sth?)

Reply:
Done.